README.md

```diff
@@ -203,7 +203,7 @@ easy to fork and contribute any changes back upstream.
    Please make sure `eval "$(pyenv init -)"` is placed toward the end of the shell
    configuration file since it manipulates `PATH` during the initialization.
     ```sh
-    $ echo 'eval "$(pyenv init -)"' >> ~/.bash_profile
+    $ echo '[[ -x $PYENV_ROOT/bin/pyenv ]] && eval "$($PYENV_ROOT/bin/pyenv init -)"' >> ~/.bash_profile
```

Comment:
1. The existence of `pyenv` executable doesn't mean we can skip `pyenv-init`. We can skip `pyenv-init` only if it has already source'd (=~ `pyenv` function exists in current shell)
1. `$PYENV_ROOT` may not be declared unless `pyenv-init` has source'd. It'd be better to use `pyenv root` since it must work as long as `$PATH` has set up

If you really want to avoid sourcing pyenv multiple times, what you really need to do must be follows. I'm not sure if it worth doing this like this though.

```sh
if ! declare -f pyenv 1>/dev/null 2>&1; then
  eval "$(pyenv init -)"
fi
```

Reply:
@yyuu 
It is not about avoiding sourcing it multiple times, but in case pyenv is not installed, e.g. when using the same dotfiles somewhere else (?!).

Reply:
That’s exactly what I wanted to say! I created this pull request because an error about `pyenv` occurs when I use my dotfiles in an environment that `pyenv` isn't installed.

Reply:
If so, what you need to do before `pyenv init` is to check the availability of `pyenv` command. I think relying on `PYENV_ROOT` here could not be a good idea since not all people are setting it.

```sh
if command -v pyenv 1>/dev/null 2>&1; then
  eval "$(pyenv init -)"
fi
```